bin/v-add-letsencrypt-domain

```diff
@@ -357,7 +357,7 @@ for auth in $authz; do
                 fi
             fi
             debug_log "Abort Step 5" "=> Wrong status"
-            check_result $E_CONNECT "Let's Encrypt validation status $status. Details: $details"
+            check_result $E_CONNECT "Let's Encrypt validation status $status for ($domain). Details: $details"
```

Comment:
If we want to keep the same style in error message remove "for"

Reply:
you're right, done :D